src/utils/formValidationOptions.ts

```diff
@@ -29,6 +38,14 @@ export const textFieldValidation: (
     value: maxLength,
     message: `${name} cannot be longer than ${maxLength} characters.`,
   },
+  ...(pattern
+    ? {
+        pattern: {
+          value: pattern,
+          message: `${name} must be a valid`,
```

Comment:
I think we will need to do a slightly better job of indicating to the user what it means for the handle to be `valid`. We don't want them to have to guess what works and what doesn't. So I guess it'd be best if we allowed passing a custom message to display in this case. (I guess we still can use the field name to stay consistent)

Reply:
I'd like the message to be something like this: `Member name may contain only lowercase letters, numbers and underscores`